decidim-core/app/views/pages/home/_statistics.html.erb

```diff
@@ -16,6 +16,30 @@
               <span class="home-pam__number processes-count"><%= participatory_processes.count %></span>
             </div>
           </div>
+          <div class="home-pam__highlight">
```

Comment:
Could we display all the published proposal's component's stats instead of hardcoding them on the front page?